src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -0,0 +1,169 @@
+    --pf-c-jump-links__link--focus--before--BorderLeftWidth: var(--pf-c-jump-links--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-jump-links__item--m-current__link--before--BorderTopWidth: var(--pf-c-jump-links--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-jump-links__item--m-current__link--before--BorderLeftWidth: var(--pf-c-jump-links--m-vertical__item--m-current__link--before--BorderLeftWidth);
+    --pf-c-jump-links__list--FlexDirection: var(--pf-c-jump-links--m-vertical__list--FlexDirection);
+
+    flex-direction: column;
+  }
+}
+
+.pf-c-jump-links__list {
+  position: relative;
+  display: flex;
+  flex-direction: var(--pf-c-jump-links__list--FlexDirection);
+  padding-top: var(--pf-c-jump-links__list--PaddingTop);
+  padding-right: var(--pf-c-jump-links__list--PaddingRight);
+  padding-bottom: var(--pf-c-jump-links__list--PaddingBottom);
+  padding-left: var(--pf-c-jump-links__list--PaddingLeft);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    pointer-events: none;
+    content: "";
+    border: solid var(--pf-c-jump-links__list--before--BorderColor);
+    border-width: var(--pf-c-jump-links__list--before--BorderTopWidth) var(--pf-c-jump-links__list--before--BorderRightWidth) var(--pf-c-jump-links__list--before--BorderBottomWidth) var(--pf-c-jump-links__list--before--BorderLeftWidth);
+  }
+}
+
+.pf-c-jump-links__link {
+  position: relative;
+  display: flex;
+  flex: 1;
+  padding-top: var(--pf-c-jump-links__link--PaddingTop);
+  padding-right: var(--pf-c-jump-links__link--PaddingRight);
+  padding-bottom: var(--pf-c-jump-links__link--PaddingBottom);
+  padding-left: var(--pf-c-jump-links__link--PaddingLeft);
+  border: 0;
+  outline-offset: var(--pf-c-jump-links__link--OutlineOffset);
+
+  &:hover {
+    --pf-c-jump-links__link-text--Color: var(--pf-c-jump-links__link--hover__link-text--Color);
+  }
+
+  &:focus {
+    --pf-c-jump-links__link-text--Color: var(--pf-c-jump-links__link--focus__link-text--Color);
+    --pf-c-jump-links__link--before--BorderTopWidth: var(--pf-c-jump-links__link--focus--before--BorderTopWidth);
+    --pf-c-jump-links__link--before--BorderLeftWidth: var(--pf-c-jump-links__link--focus--before--BorderLeftWidth);
+    --pf-c-jump-links__link--before--BorderColor: var(--pf-c-jump-links__link--focus--before--BorderColor);
+  }
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border-color: var(--pf-c-jump-links__link--before--BorderColor);
+    border-style: solid;
+    border-width: var(--pf-c-jump-links__link--before--BorderTopWidth) var(--pf-c-jump-links__link--before--BorderRightWidth) var(--pf-c-jump-links__link--before--BorderBottomWidth) var(--pf-c-jump-links__link--before--BorderLeftWidth);
+  }
+}
+
+// stylelint-disable
+.pf-c-jump-links__item {
+  list-style: none;
```

Comment:
is this necessary?

Reply:
not anymore not that subsections is removed